modules/ROOT/pages/split-aggregate-scope.adoc

```diff
@@ -0,0 +1,72 @@
+= Split Aggregate Scope
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, studio, mule, split, aggregate, scope
+
+The Split Aggregate scope splits a collection, and processes each part with the components within it in parallel (subject to any limitation in concurrency that had been configured).
+After all messages have been processed, they will be aggregated before continuing with the flow, in the same order they were before splitting the defined collection. 
+
+The Split Aggregate scope provides fields described in this table:
+
+[%header,cols="1,3"]
+|===
+|Attribute |Description
+|Collection (`collection`) | Specifies the expression that define the collection of parts to be processed in parallel. By default, it will use the incoming payload.
+|Timeout (`timeout`) | Specifies the timeout for each parallel route. The default behaviour is that of no timeout set.
+|Max Concurrency (`maxConcurrency`) | Specifies the maximum level of parallelism that will be used by this router. By default, all routes will run in parallel.
+|Target Variable (`target`) | Specifies the variable where to save the processed payload. 
+|Target Value (`targetValue`) | Specifies an expression to evaluate against the operation's output value. The outcome of this expression will be stored in the target variable. 
+|===
+
+This XML example adds to every element in the collection the string `"-result"`: 
+
+----
+<split-aggregate>
+	<collection>
+		['apple', 'banana', 'orange']
+	</collection>
+	<set-payload value="#[payload ++ '-result']"/>
+</split-aggregate>
+<!-- payload will now be ['apple-result', 'banana-result', 'orange-result']
+----
+
+The example above is equivalent to: 
+
+----
+<split-aggregate collection="#[['apple', 'banana', 'orange']]">
+	<set-payload value="#[payload ++ '-result']"/>
+</split-aggregate>
+----
+
+The next example shows that the variables can also be aggregated:
+----
+<split-aggregate collection="#[['apple', 'banana', 'orange']]">
+	<choice>
+		<when expression="#[payload == 'apple']">
+			<set-variable variableName="var1" value="newValue"/>
+			<set-variable variableName="var2" value="val1"/>
+		</when>
+		<when expression="#[payload == 'banana']">
+			<set-variable variableName="var2" value="val2"/>
+		</when>
+		<otherwise>
+			<set-variable variableName="var2" value="val3"/>
+		</otherwise>
+	</choice>
+</split-aggregate>
+<!-- After aggregation, the variables will be: {var1: "newValue", var2: ["val1, val2, val3"]} -->
+----
+
+=== Differences between Split Aggregate scope and For Each scope
+
+There is an obvious similarity between the Split Aggregate scope and the xref:for-each-scope-concept.adoc[For Each scope]: both split the defined collection and process each part it with the components within the scope. Also, in both cases each route runs with the same initial context.
```

Comment:
`Also, in both cases each` -> `Also, in both cases, each`

Reply:
`and process each part` -> `and process each element` 
but I would rephrase it a little bit, is quite confusing,

Reply:
`Also, in both cases each route runs with the same initial context.` Nobody will understand what is the `initial context`